TheraLang.Tests/Setups/AuthenticationServiceSetups.cs

```diff
@@ -0,0 +1,78 @@
+﻿using Common.Configurations;
+using Microsoft.AspNetCore.Http;
+using Microsoft.Extensions.Options;
+using Moq;
+using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Security.Claims;
+using TheraLang.BLL.Services;
+using TheraLang.DAL.Entities;
+using TheraLang.DAL.UnitOfWork;
+using TheraLang.Tests.Mocks;
+
+namespace TheraLang.Tests.Setups
+{
+    public class AuthenticationServiceSetups
```

Comment:
Maybe create some generic base setup class where base type repository and uow will be created, cause this code is in every setup class